demo/demo.md

```diff
@@ -65,6 +67,8 @@ In these examples the trigger can be external to the scope of the popover compon
 
 NOTE: The popover element is hidden visually, but not set to `display: none` for accessibility reasons. This is important to know in this setup because the `auro-popover` element may take up physical space depending on your layout solution.
 
+Note that in these examples the popover content will *not* persist as the user moves their mouse off of the trigger and over the popover content.
```

Comment:
Why is there a difference between this and the slot use case?

Reply:
By moving the hide/show trigger behavior onto the `auro-popover` element itself rather than the trigger element we essentially get the content hover persistence behavior for free in a manner of speaking.

With a disconnected trigger it's possible to have more of a significant gap in between the trigger and the content depending on how it's implemented. I can't say that anyone would necessarily choose to do that - but nothing stops them from doing it.

Extending the persistence to cover a disconnected trigger would mean something more involved - probably using a timeout to persist the content for a small amount of time after the cursor leaves the trigger and then removing the timeout if the cursor moves over the content before it expires.

This felt like a heavy handed approach, where creating an expectation that persistence meant you had to use a nested trigger seemed more reasonable. What's your opinion on that, @geoffrich?

Reply:
I am thinking that there is an opportunity here to make this demo more clear. 

1. Popover use cases
2. Using slots, internal to the component
3. Add space around popover
4. Using disconnected elements
5. Do's and don'ts

I don't think we need to add the `works here and not here` lines. When this PR is complete, I want to make updates to all the content on this demo page. 

The thing we can do here is to draw more attention to the idea that if a persisted experience is wanted for disconnected elements, then the `sticky` feature is better suited.

Reply:
Great idea @blackfalcon